packages/contracts/contracts/protocol/lendingpool/LendingPoolConfigurator.sol

```diff
@@ -111,6 +108,62 @@ contract LendingPoolConfigurator is
     }
 
 
+    /**
+     * @dev Verifies tranche for more privileges
+     * @param trancheId the tranche that is verified
+     **/
+    function verifyTranche(
+        uint64 trancheId
+    ) external onlyGlobalAdmin {
+        require(trancheId<totalTranches, Errors.INVALID_TRANCHE);
+        require(!pool.getTrancheParams(trancheId).verified, Errors.ALREADY_VERIFIED);
+        // TODO: enforce possible staking of VMEX
+
+        // loop through all reserves in this tranche, and set the local values of LTV, threshold, bonus, and borrow factor
+        address[] memory reserves = pool.getReservesList(trancheId);
+        // TODO: possible OOG?
+        for(uint i = 0;i<reserves.length; ++i) { //initialize with initial global asset params
```

Comment:
uint256 cachedLength = reserves.length;
for(uint256 i; i < cachedLength; ) {
 unchecked { ++i }
}

Reply:
We can generalize this for all loops.